src/main/java/net/mateakademy/mapping/ProducerMapper.java

```diff
@@ -0,0 +1,31 @@
+package net.mateakademy.mapping;
+
+
+import net.mateakademy.dto.Producer;
+import net.mateakademy.model.ProducerEntity;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ProducerMapper {
+
+    public Producer mapProducerEntityToProducer(ProducerEntity entity) {
+        return new Producer()
```

Comment:
better use builders in mapper class
add lombok annotation builder to needed classes